community/kernel/src/main/java/org/neo4j/kernel/info/SystemDiagnostics.java

```diff
@@ -160,9 +160,17 @@ void dump( Logger logger )
                 if ( loader instanceof URLClassLoader )
                 {
                     URLClassLoader urls = (URLClassLoader) loader;
-                    for ( URL url : urls.getURLs() )
-                        if ( "file".equalsIgnoreCase( url.getProtocol() ) )
-                            paths.put( url.toString(), pathValue( paths, "loader." + level, url.getPath() ) );
+                    URL[] classLoaderUrls = urls.getURLs();
+                    if ( classLoaderUrls != null )
+                    {
+                        for ( URL url : urls.getURLs() )
+                            if ( "file".equalsIgnoreCase( url.getProtocol() ) )
+                                paths.put( url.toString(), pathValue( paths, "loader." + level, url.getPath() ) );
```

Comment:
The loop can use the `classLoaderUrls` variable, instead of constructing the array again.

Also, the formatting can be improved here. Otherwise, the forward merger will have to do it to make it build again.